kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -335,6 +335,9 @@
           this.hasMultipleFacilities || get(this.selectedFacility, 'dataset.preset') !== 'informal'
         );
       },
+      isNextValid() {
```

Comment:
Related to the above comment, I think I would just keep the `busy` part out of this property, or maybe rename it so it's more general, something  to the effect of "next button is enabled"